src/Core/src/Handlers/IViewHandler.cs

```diff
@@ -2,16 +2,38 @@
 
 namespace Microsoft.Maui
 {
+	/// <summary>
+	/// Defines members that view handlers should implement to provide mapping virtual views to platform views.
+	/// </summary>
 	public interface IViewHandler : IElementHandler
 	{
+		/// <summary>
+		/// Gets or sets whether the <see cref="IElementHandler.PlatformView"/> has a view that it is contained in.
+		/// </summary>
 		bool HasContainer { get; set; }
```

Comment:
Should we add a remark on when this is applicable? What should the consuming developer know about this?

Reply:
What controls _don't_ have a container? If the list is small we can just add a small remark saying something like "X, Y, and Z will never have a container"